custom/enikshay/management/commands/data_dumps_dstb_episodes.csv

```diff
@@ -1 +1,3 @@
Case Definition,Primary Case,self,case_definition
Initiated on first line treatment,Primary Case,self,treatment_initiated
Supported by RNTCP?,N/A,N/A,N/A
Current Treatment status,N/A,N/A,N/A
Treatment (Initiation) Status,Primary Case,self,treatment_status
Date of treatment initiation,Primary Case,self,treatment_initiation_date
Type of treatment adherence,Primary Case,self,adherence_type_choice
Type of treatment adherence - (Other - Specify),Primary Case,self,adherence_type_other_detail
Specify ICT System,Primary Case,self,adherence_type_ict_choice
Other ICT System,Primary Case,self,adherence_type_ict_other_detail
Treatment Supporter Designation,Primary Case,self,treatment_supporter_designation
Treatment Supporter First Name,Primary Case,self,treatment_supporter_first_name
Treatment Supporter Last Name,Primary Case,self,treatment_supporter_last_name
Treatment Supporter Mobile Number,Primary Case,self,treatment_supporter_mobile_number
Weight,Primary Case,self,weight
Height,Primary Case,self,height
Dosage Frequency,Primary Case,self,adherence_schedule_id
Drug Formulations,Primary Case,self,drug_formulations
Loose Drugs Dosage,Primary Case,self,dosage_loose_drugs
Weight Band,Primary Case,self,weight_band
"Drug Packaging, if any",Primary Case,self,drug_packaging
"FDC Dosage per Day, if any",Primary Case,self,dosage_fdc
HIV status,N/A,N/A,N/A
"Date of HIV Test, if Reactive & Non - Reactive",N/A,N/A,N/A
"PID Number, if any",N/A,N/A,N/A
CPT Initiated?,N/A,N/A,N/A
"Date of CPT Initiation?, if yes",N/A,N/A,N/A
ART Initiated?,N/A,N/A,N/A
Date of ART Initiation?,N/A,N/A,N/A
"ART Number, if any",N/A,N/A,N/A
"ART Centre Name, if any",N/A,N/A,N/A
"HIV Testing offered?, if HIV status is Unknown",N/A,N/A,N/A
Diabetic Status,N/A,N/A,N/A
RBS Level,N/A,N/A,N/A
FBS Level,N/A,N/A,N/A
Initiated on Anti-diabetes treatment?,N/A,N/A,N/A
Date of anti-diabetes treatment initiation,N/A,N/A,N/A
Anti diabetes treatment program number,N/A,N/A,N/A
"Diabetes testing offered?, if Diabetes Status is Unknown",N/A,N/A,N/A
Other Comorbidity,N/A,N/A,N/A
"Specify Other Co-morbidity, if selected as Yes",N/A,N/A,N/A
Current Tobacco user?,N/A,N/A,N/A
Tobacco Type?,N/A,N/A,N/A
Linked to cessation program,N/A,N/A,N/A
History of Alcohol intake,N/A,N/A,N/A
Linked for deaddiction Program?,N/A,N/A,N/A
Has Nikshay ID?,N/A,N/A,N/A
Enter Nikshay ID,N/A,N/A,N/A
Has TB Notification Number for current episode,N/A,N/A,N/A
Enter TB Number,Primary Case,self,tb_number
Marital Status,N/A,N/A,N/A
Marital Status (Other - Specify),N/A,N/A,N/A
Socio Economic Status,N/A,N/A,N/A
Name & Address of Contact Person,N/A,N/A,N/A
Mobile/ Landline Number of Contact Person,N/A,N/A,N/A
Predominant Symptoms,Primary Case,self,predominant_symptom
"Duration of symptoms in Days, if any",Primary Case,self,symptom_duration
History of ATT greater than one month?,N/A,N/A,N/A
Number of health care providers visited before diagnosis for current episodes,N/A,N/A,N/A
"Months of Treatment undertaken
```

Comment:
I notice that the "Type" column is a new addition to these spreadsheets - did you update the base command to account for it?  I guess this uses `csv.DictReader`, so maybe that's not relevant, huh?